src/seedu/addressbook/data/AddressBook.java

```diff
@@ -58,6 +58,12 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sorts all persons from the address book.
+     */
+
+    public void sorted()  { allPersons.sort();}
```

Comment:
The placement of your code is violating the coding convention. Please add logic on new line. Refer to java coding conventions for CS2113/T for more information.